src/test/java/org/jabref/logic/importer/fetcher/ArXivTest.java

```diff
@@ -144,19 +148,18 @@ void searchEntryByPartOfTitleWithAcuteAccent() throws Exception {
 
     @Test
     void searchEntryByOldId() throws Exception {
-        BibEntry expected = new BibEntry();
-        expected.setType(StandardEntryType.Article);
-        expected.setField(StandardField.AUTHOR, "H1 Collaboration");
-        expected.setField(StandardField.TITLE, "Multi-Electron Production at High Transverse Momenta in ep Collisions at HERA");
-        expected.setField(StandardField.DATE, "2003-07-07");
-        expected.setField(StandardField.ABSTRACT, "Multi-electron production is studied at high electron transverse momentum in positron- and electron-proton collisions using the H1 detector at HERA. The data correspond to an integrated luminosity of 115 pb-1. Di-electron and tri-electron event yields are measured. Cross sections are derived in a restricted phase space region dominated by photon-photon collisions. In general good agreement is found with the Standard Model predictions. However, for electron pair invariant masses above 100 GeV, three di-electron events and three tri-electron events are observed, compared to Standard Model expectations of 0.30 \\pm 0.04 and 0.23 \\pm 0.04, respectively.");
-        expected.setField(StandardField.EPRINT, "hep-ex/0307015");
-        expected.setField(StandardField.FILE, ":http\\://arxiv.org/pdf/hep-ex/0307015v1:PDF");
-        expected.setField(StandardField.EPRINTTYPE, "arXiv");
-        expected.setField(StandardField.EPRINTCLASS, "hep-ex");
-        expected.setField(StandardField.KEYWORDS, "hep-ex");
-        expected.setField(StandardField.DOI, "10.1140/epjc/s2003-01326-x");
-        expected.setField(StandardField.JOURNALTITLE, "Eur.Phys.J.C31:17-29,2003");
+        BibEntry expected = new BibEntry(StandardEntryType.Article);
+        expected.withField(StandardField.AUTHOR, "H1 Collaboration")
```

Comment:
```suggestion
        .withField(StandardField.AUTHOR, "H1 Collaboration")
```

Please chain it, too.